.eslintrc.json

```diff
@@ -46,7 +46,7 @@
       }
     ],
     "no-shadow": "off",
-    "jsx-a11y/label-has-associated-control": "warn",
```

Comment:
I agree about keeping a11y checks project-wide, but this particular case is becoming a bit of an annoyance. Somehow it causes warnings for things that should be valid, plus there is a bit of incompatibility with the visual framework. If I remember correctly the issue is:
```html
<label htmlFor="input_id">The input</label>
<input id="input_id" />
```
is causing the warning even when the [documentation](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md#case-the-label-is-a-sibling-of-the-control) says it's right.

This doesn't triggers the warning:
```html
<label htmlFor="input_id">
  The input
  <input id="input_id" />
</label>
```
But the css rule in the vs framework is defined with sibling elements so the styling gets a bit mangled.

I suppose the correct thing to do is to follow up with the respective dependencies.

Reply:
Good followup. If I understand right from some GitHub issue comments, this is being marked as invalid because of our airbnb-inherited lint config deciding that sibling label/controls are invalid. I believe we need to override something about about this part: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react-a11y.js#L121
But I don't know exactly what...